test/functional/feature_llmq_simplepose.py

```diff
@@ -85,6 +88,32 @@ def force_old_mn_proto(self, mn):
         self.reset_probe_timeouts()
         return False
 
+    def test_pose_cooldown(self):
+        # Pretend that we are still in PoSe cooldown period
+        for mn in self.mninfo:
+            self.stop_node(mn.node.index)
+            os.remove(os.path.join(mn.node.datadir, self.chain, 'mncache.dat'))
+            self.start_masternode(mn, ["-vbparams=v17:%s:%s" % (self.mocktime, self.mocktime + 31536000)])
```

Comment:
What is 31536000?

Reply:
seconds in a year

Reply:
Replaced the magic number with corresponding calculations, reworked the test and squashed it all into bdfe952.